buildout.cfg

```diff
@@ -23,10 +28,10 @@ eggs =
   zc.buildout[test]
   zc.recipe.egg
   coverage
-initialization =
-  import os
-  if 'COVERAGE_PROCESS_START' not in os.environ: os.environ['COVERAGE_PROCESS_START'] = os.path.abspath('../../.coveragerc')
-  if os.getenv('COVERAGE_PROCESS_START'): import coverage; coverage.process_startup()
+#initialization =
+#  import os
+#  if ('TOX_ENV_NAME' in os.environ) and ('COVERAGE_PROCESS_START' not in os.environ): os.environ['COVERAGE_PROCESS_START'] = os.path.abspath('../../.coveragerc')
+#  if os.getenv('COVERAGE_PROCESS_START'): import coverage; coverage.process_startup()
```

Comment:
Remove entirely?

Reply:
I kept it for now until I manage to run coverage properly again.